bluebutton-data-model-rif/src/main/java/gov/hhs/cms/bluebutton/data/model/rif/RifDataloadHistory.java

```diff
@@ -0,0 +1,167 @@
+package gov.hhs.cms.bluebutton.data.model.rif;
+
+/**
+ * Entity that stores when each RIF dataload file is loaded to the PostgreSQL database
+ */
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+@Entity
+@IdClass(RifDataloadHistory.PrimaryKey.class)
+@Table(name = "`RifDataloadHistory`")
+public class RifDataloadHistory {
+
+	@Id
+	@Column(name = "`recordType`", nullable = false)
+	private String recordType;
+
+	@Id
+	@Column(name = "`createUpdateTimestamp`", nullable = false)
+	private Timestamp createUpdateTimestamp;
+
+	@Id
+	@Column(name = "`sequenceId`", nullable = false, columnDefinition = "numeric")
+	private int sequenceId;
+
+	@Column(name = "`dataloadFilename`", nullable = false, length = 64)
+	private String dataloadFilename;
+
+	public RifDataloadHistory() {
+		super();
+	}
+
+	/**
+	 * @param recordType
+	 * @param createUpdateTimestamp
+	 * @param sequenceId
+	 */
+	public RifDataloadHistory(String recordType, Timestamp createUpdateTimestamp, int sequenceId) {
+		super();
+		this.recordType = recordType;
+		this.createUpdateTimestamp = createUpdateTimestamp;
+		this.sequenceId = sequenceId;
+	}
+
+	public String getRecordType() {
+		return recordType;
+	}
+
+	public void setRecordType(String recordType) {
```

Comment:
Are all these setters ever used (instead of the constructor, that is)?

Reply:
Yes in RifLoader